packages/cli/src/lib/Cli.ts

```diff
@@ -36,10 +36,11 @@ export class Cli {
       throw new Error(`Current Node.js version is ${process.versions.node}.` +
           ' Node.js version 10 or above is required to run bubblewrap.');
     }
-    const config = await loadOrCreateConfig();
-
     const parsedArgs = minimist(args);
 
+    const config = await loadOrCreateConfig(
```

Comment:
If we revert `loadOrCreateConfig()`, we should be able to call `loadOrCreateConfig(null, null, parsedArgs.config`). So, the 1st and 2nd parameters get the default value and the last uses `parsedArgs.config`. If `parsedArgs.config` is null, it will use the default value.

Reply:
Well, since there aren't named arguments, and they were cases where some were used and some not, I figured a dictionary would be cleaner than `loadOrCreateConfig(undefined, undefined, parsedArgs.config)`, but I don't really care one way or another on the style.